### PR TITLE
Update channels.csv

### PR DESCRIPTION
**Pull request by hydrationn**
### Description
- This PR updates the SBS channel information in the channel database, correcting existing data and enhancing details such as the logo, website, and broadcast categories.

### Related issue
- Edit SBS channel information #14425

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -25327,6 +25327,7 @@ SBNTV.us,SBN TV,,,,US,,,c/US,eng;spa,religious,FALSE,,,,https://www.sonlifetv.co
 SBNTVGlobal.us,SBN TV Global,,,,US,,,r/INT,eng;por;spa,religious,FALSE,,,,https://www.sonlifetv.com/,https://i.imgur.com/NJu6oYB.png
 SBNTVInternational.us,SBN TV International,,,,US,,,r/INT,eng;spa;som,religious,FALSE,,,,https://www.sonlifetv.com/,https://i.imgur.com/NJu6oYB.png
 SBS.us,SBS,Shinniigaa Broadcasting Service,,,US,,,r/AFR,som,news,FALSE,,,,,https://i.imgur.com/bOHMrky.png
+SBS.kr,SBS,Seoul Broadcasting System,,,,KR,,,c/KR,kor,general,FALSE,,,,https://www.sbs.co.kr/,https://imgur.com/a/uHlt1Lv
```
Comment (freearhey) [inline data/channels.csv:25330]:
```suggestion
SBS.kr,SBS,Seoul Broadcasting System,,,,KR,,,c/KR,kor,general,FALSE,,,,https://www.sbs.co.kr/,https://imgur.com/4LHX71N.png
```

**Review by freearhey — CHANGES_REQUESTED** [review]

**Review by freearhey — CHANGES_REQUESTED** [review]
To add information about SBS channel you need to edit the line with ID `SBSTV.kr` instead of creating a new entry.

**Review by freearhey — APPROVED** [review]

**Comment by hydrationn** [comment]
@ProPontifex,
I’ve updated the code based on the feedback provided earlier regarding issue #14343. Could you please review the changes when you have time? Let me know if there are any additional adjustments needed. Thank you!

File: data/channels.csv (hunk 1 above)
Comment (BellezaEmporium) [reply data/channels.csv:25330]: i.imgur.com*

**Comment by BellezaEmporium** [comment]
Image link is broken, I believe it's "i.imgur.com", not "imgur.com"

**Review by freearhey — APPROVED** [review]

**Comment by hydrationn** [comment]
> 이미지 링크가 끊어졌습니다. "imgur.com"이 아니라 "i.imgur.com"인 것 같습니다.

Thank you for your feedback, I've done the modification!

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**